src/logger/logger.js

```diff
@@ -11,7 +11,7 @@ const WinstonCloudwatch = require("winston-cloudwatch")
 const { config } = require("@config/config")
 
 // Env vars
-const NODE_ENV = config.get("env")
+const NODE_ENV = config.get("env").toLowerCase()
```

Comment:
Eh wait why are we not directly changing this ah in prod/staging ah?

Reply:
not in favour of doing this here - should either do directly in the respective env as per ^ or we should normalise it in the convict function to ensure that it's case agnostic.

more in favour of the latter cos it'll save devs from having to remember if it's upper or lower

Reply:
yeah that makes sense, have removed toLowerCase in https://github.com/isomerpages/isomercms-backend/pull/659/commits/0b7f6c13cdab0dd43a3ada79518a45bf7357d9bb